src/server/plugins/engine/components/ComponentCollection.ts

```diff
@@ -232,11 +232,7 @@ export class ComponentCollection {
     return list
   }
 
-  getViewModel(
-    payload: FormPayload,
-    errors?: FormSubmissionError[],
-    conditions?: FormModel['conditions']
```

Comment:
We never pass in the 3rd `conditions` param so I've removed it

Over in PR https://github.com/DEFRA/forms-runner/pull/646 the 3rd param comes back as `query` to pick up `?force` direct access